docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,12 @@
+## Title for RFC
+
+#### Abstract
+
+
+##### Architecture Changes
```

Comment:
Insufferable nitpick: is this supposed to be a different header size than the others (5 "#"s vs. 4)